### PR TITLE
Deployer updates

### PR DESCRIPTION
**Pull request by lmars**
The main change here is to make the deployer log more verbosely, as currently it only really logs on error. I also made the log messages more consistent by always using lower case, and starting error messages with the word "error" to make the logs easier to scan down.

### CODE REVIEW

File: controller/deployer/strategies/one-by-one.go [hunk 1]
```diff
@@ -7,36 +7,40 @@ import (
...
 	if err != nil {
-		log.Error("Failed fetching the old formation", "at", "get_formation", "err", err)
+		olog.Error("error getting old formation", "err", err)
 		return err
 	}
 
-	oldFormation := f.Processes
-	newFormation := map[string]int{}
+	oldProcesses := f.Processes
+	newProcesses := map[string]int{}
```
Comment (titanous) [inline controller/deployer/strategies/one-by-one.go]: This should use `make`

File: test/test_deployer.go [hunk 2]
```diff
@@ -41,14 +41,14 @@ func (s *DeployerSuite) createDeployment(t *c.C, process, strategy string) *ct.D
 }
 
 func waitForDeploymentEvents(t *c.C, stream chan *ct.DeploymentEvent, expected []*ct.DeploymentEvent) {
-	// wait for an event with no release to mark the end of the deployment,
-	// collecting events along the way
-	events := []*ct.DeploymentEvent{}
+	debugf(t, "waiting for %d deployment events", len(expected))
+	actual := []*ct.DeploymentEvent{}
```
Comment (titanous) [inline test/test_deployer.go]: This should use make and have a capacity of `len(expected)`

**Comment by lmars** [comment]
@titanous comments addressed

**Comment by titanous** [comment]
LGTM

**Pull request merged**